content/md/imports-exports/tailored-export.md

```diff
@@ -0,0 +1,86 @@
+---
+id: tailored-export
+themes: import-export-data
+title: "Explore new possibilities with a tailored export"
+popular: false
+ee-only: true
+related: exports, monitor-jobs, access-rights-on-imports-exports
+---
+
+# Overview
+
+This feature consists in a new export profile, available only for our Enterprise Edition or Growth Edition users.
+It helps you to export your product information in flat files, in the ways you want.  
+For instance, let's assume your retailers or distributors need the same product information from you in flat file, but they both want different structures:  
+* Retailer A wants an export with columns named: "Identifier", "Product Name", "Description" and "Weight (cm)".
+* Retailer B wants an export with columns named: "Name in french", "Name in english", "Description 1", "ID" and "Weight (mm)".  
+
+It means you need to export your information thanks to the classic product export, and to manually modify your spreadsheet to reorder the columns, change their name, convert the measurement and other low value-added tasks.  
+Now you can easily create different Tailored Export profiles to match the requirements of most of your retailers or distributors!
+
+# Create a tailored product export profile
+
+To create a product export:
+1. Go to `Exports`
+1. Click on `Create export profile`
+1. Type a unique export code and a label, then select the job `Tailored product export` or `Tailored product model export`. You can select either CSV or XLSX for each job.
+![Create a new product export profile](../img/TailoredExport_Create-export-profile.png)
+1. Save your new export profile
+
+# Edit a tailored product export profile
+
+Once you've created your tailored export profile, or if you want to edit an existing tailored product export profile:
+1. Go to `Exports`
+1. Click on the tailored export profile you want to edit
+1. Click on `Edit` located in the top right corner of the screen  
+
+## Discover the different tabs of this profile
+
+As in all export profiles, the tailored export profile displays different tabs where you can configure it. Here's the list of the different tabs:
+
+* `Properties`: Define the label of your export profile
+* `Permissions`: Set permissions to allow specific user groups to execute or edit your export profile (Enterprise Edition only).
+* `Global settings`: Define the max number of lines per file you want, the field delimiter (CSV format only) and the field enclosure character (CSV format only).
+* `Product selection`: Use the provided filters to decide which products will be exported (see [Focus on "Product selection" section](tailored-export.html#focus-on-product-selection-section)").
+* `Export structure`: Build your export structure (see [Focus on "Export structure" section](tailored-export.html#focus-on-export-structure)").
+* `History`: Display the modifications made by any user on your export profile.
```

Comment:
```suggestion
```

Reply:
We decided to keep ths with Camille, so that our new users are still onboarded in the good way, with an explaination for every tab.